.kitchen.yml

```diff
@@ -60,6 +64,31 @@ suites:
           checkpoint: true
           switch_user: yes
 
+- name: attributes
```

Comment:
Hmm... as there is not much difference between test suites in general, what do you think about rewriting every test suite in this kitchen YAML to use anchors from the default suite?